### DISCUSSION TITLE
【求助】绳网那个默认的封面有人能帮忙发一下嘛

### DISCUSSION THREAD

**Discussion by ShinyuanJ** [share121/inter-knot#2492] [其他]
非常感谢！

**Comment by isle200inf** [comment]
![default-cover oB27RXoN](https://github.com/user-attachments/assets/93a1099f-0abc-4882-8537-8a571274a33d)
![default-cover oB27RXoN](https://github.com/user-attachments/assets/4c2db51d-c1a3-4c56-b607-8a1ea32e668b)

**Reply by isle200inf** [reply]
是这个吗
